_posts/2020-06-11-richard-fitzjohn.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Should this commit be here?

Reply:
oh I don't know how that slipped in. That's the previous episode.